pkg/email/send_email.go

```diff
@@ -8,6 +8,11 @@ import (
 	"gopkg.in/gomail.v2"
 )
 
+type Email interface {
```

Comment:
This is not what I meant in my previous comment.
- Interface should describe the operation, like: EmailSender
- It would probably include just a method and the one that passes the implementation should pass one that use or not use limit.
- The mailer implementation can hold two ways to send email - with and without limit, but in our configuration we configure one mail - so from the user perspective we configure it.  A preferred way so having two methods in mailer is good for testing and internal implementation - but not for reflecting the mailer behavior.